src/components/accounts/SetupSeedPhrase.js

```diff
@@ -233,17 +234,18 @@ class SetupSeedPhrase extends Component {
                                         <h1><Translate id='setupSeedPhraseVerify.pageTitle'/></h1>
                                         <h2><Translate id='setupSeedPhraseVerify.pageText'/></h2>
                                         <SetupSeedPhraseVerify
-                                            enterWord={this.state.enterWord}
-                                            wordId={this.state.wordId}
+                                            enterWord={enterWord}
+                                            wordId={wordId}
                                             handleChangeWord={this.handleChangeWord}
                                             handleStartOver={this.handleStartOver}
-                                            mainLoader={this.props.mainLoader || this.state.submitting}
-                                            localAlert={this.state.localAlert}
+                                            mainLoader={this.props.mainLoader || submitting}
+                                            localAlert={localAlert}
                                             globalAlert={this.props.globalAlert}
                                             onRecaptchaChange={this.handleRecaptchaChange}
                                             ref={(ref) => this.recaptchaRef = ref}
-                                            isNewAccount={this.state.isNewAccount}
+                                            isNewAccount={isNewAccount}
                                             onSubmit={this.handleOnSubmit}
+                                            isLinkDrop={parseFundingOptions(this.props.location.search)}
```

Comment:
For type consistency, and to avoid the 'don't surprise me' principle, it would be best if we passed in a boolean for `isLinkDrop` here rather than an object that we check in a truesy way internally.  Since you made `parseFundingOptions()` return `null` we can just update this to `parseFundingOptions(this.props.location.search) !== null` safely :+1: